test/server/cards/06_SEC/units/GalenErsoYoullNeverWin.spec.ts

```diff
@@ -0,0 +1,1325 @@
+describe('Galen Erso - You\'ll Never Win', function() {
+    integration(function(contextRef) {
+        it('should be playable using Plot and should name a card', async function () {
+            await contextRef.setupTestAsync({
+                phase: 'action',
+                player1: {
+                    leader: 'luke-skywalker#faithful-friend',
+                    resources: ['galen-erso#youll-never-win', 'wampa', 'wampa', 'wampa', 'wampa', 'wampa'],
+                    deck: ['moisture-farmer']
+                }
+            });
+
+            const { context } = contextRef;
+
+            context.player1.clickCard(context.lukeSkywalker);
+            context.player1.clickPrompt('Deploy Luke Skywalker');
+
+            expect(context.player1).toHavePassAbilityPrompt('Play Galen Erso using Plot');
+            context.player1.clickPrompt('Trigger');
+            expect(context.galenErso).toBeInZone('groundArena');
+            expect(context.moistureFarmer).toBeInZone('resource');
+            expect(context.player1).toHaveExactDropdownListOptions(context.getAllNonLeaderCardTitles());
+            context.player1.chooseListOption('Luke Skywalker');
+        });
+
+        describe('Galen Erso - You\'ll Never Win\'s ability should name a card. While he is in play, named enemy leaders', function() {
+            it('should not be blanked when their title is named', async function () {
+                await contextRef.setupTestAsync({
+                    phase: 'action',
+                    player1: {
+                        hand: ['galen-erso#youll-never-win'],
+                    },
+                    player2: {
+                        leader: 'luke-skywalker#faithful-friend',
+                        hand: ['battlefield-marine'],
+                    }
+                });
+
+                const { context } = contextRef;
+
+                context.player1.clickCard(context.galenErso);
+                context.player1.chooseListOption('Luke Skywalker');
+
+                context.player2.clickCard(context.battlefieldMarine);
+                context.player1.passAction();
+                context.player2.clickCard(context.lukeSkywalker);
+                context.player2.clickPrompt('Give a shield to a heroism unit you played this phase');
+                context.player2.clickCard(context.battlefieldMarine);
+                expect(context.battlefieldMarine).toHaveExactUpgradeNames(['shield']);
+            });
+
+            it('should not be blanked when their title is named', async function () {
```

Comment:
nit: case name should indicate that the leader is deployed or is a "leader unit" to distinguish from previous test case